gateway-service/src/main/java/org/zowe/apiml/gateway/services/ServicesInfoService.java

```diff
@@ -92,8 +93,8 @@ public ServiceInfo getServiceInfo(String serviceId) {
 
     private String getBaseUrl(ApiInfo apiInfo, InstanceInfo instanceInfo) {
         ServiceAddress gatewayAddress = gatewayClient.getGatewayConfigProperties();
-        return String.format("%s://%s%s",
-                gatewayAddress.getScheme(), gatewayAddress.getHostname(), getBasePath(apiInfo, instanceInfo));
+        return UrlUtils.getUrl(gatewayAddress.getScheme(), gatewayAddress.getHostname()) +
+               getBasePath(apiInfo, instanceInfo);
```

Comment:
incorrect use of parameter, it should be hostWithPort but contains path as well, better have separate method

Reply:
I missed ")", my bad